.github/workflows/packaging.yaml

```diff
@@ -94,3 +94,28 @@ jobs:
         run: |
           ./packaging/msi/winget_manifest.sh > Tetrate.func-e.yaml
           yamllint -sd '{extends: default, rules: {line-length: disable}}' Tetrate.func-e.yaml
+
+      # This only checks installing and unistalling the Debian package (amd64) when running this job
+      # step on Linux (Ubuntu, amd64) as it is more efficient than having a new job definition
+      # (which requires to upload and download the built packages as artifacts) and do the
+      # verification there.
+      - name: "Test installing and unistalling Debian package (on Ubuntu, amd64)"
+        if: runner.os == 'Linux'
+        # Running dpkg and apt-get requires "sudo".
+        run: sudo packaging/nfpm/verify_deb.sh
+
+      # This only checks installing and unistalling the RPM package when running this job on Linux,
+      # but using container (on CentOS 8 via ghcr.io/tetratelabs/func-e-internal:centos8, x86_64) as
+      # it is more efficient than having a new job definition (which requires to upload and download
+      # the built packages as artifacts) and do the verification there.
+      - name: "Test installing and unistalling RPM package (on CentOS 8, x86_64 via ghcr.io/tetratelabs/func-e-internal:centos8)"
+        if: runner.os == 'Linux'
+        uses: docker://ghcr.io/tetratelabs/func-e-internal:centos8
+        with:
+          # Since we use the root user from within the container process when doing this, we don't
+          # require "sudo".
+          args: packaging/nfpm/verify_rpm.sh
+
+      # TODO: Verification for installing and unistalling arm64 Debian and aarch64 RPM packages.
+      # Curently it is not attempted since we need to pass the artifacts to Travis CI every time
+      # this job is finished.
```

Comment:
Since currently I don't want to be creative on this, and not sure if the creativity will be still in the scope of this change.

For example: running make dist on graviton.

Reply:
Do you have ideas on this @mathetake?

Reply:
I'm thinking to remove Travis and use `self-hosted` GHA instead. Probably in another PR.

Reply:
please don't as this is worse and docker in general breaks in many releases as well emulation setup is quite more problematic than the risk that the RPM verification will fail. What travis is doing is infinitely more simple to troubleshoot than emulation. You can feel free to use emulation on your own, but please don't pull that in here. I won't be fixing it months later when it invariably breaks due to some strange relicensing of docker or arch mismatch problem. you can notice even zipkin's build recently broke due to a docker change in emu